tests/ui/test_ui_tools.py

```diff
@@ -2772,12 +2772,15 @@ def test_handle_link(self, mocker, link, handle_narrow_called):
         assert self.handle_narrow.called == handle_narrow_called
 
     @pytest.mark.parametrize(['link', 'is_subscribed', 'stream_id',
-                              'stream_name'], [
-        (SERVER_URL + '/#narrow/stream/1-Stream-1', True, 1, 'Stream 1'),
-        (SERVER_URL + '/#narrow/stream/4-Stream-4', False, 4, None),
+                              'stream_name', 'near'], [
+        (SERVER_URL + '/#narrow/stream/1-Stream-1', True, 1, 'Stream 1', None),
+        (SERVER_URL + '/#narrow/stream/4-Stream-4', False, 4, None, None),
+        (SERVER_URL + '/#narrow/stream/1-Stream-1/near/2', True, 1, 'Stream 1',
```

Comment:
Could you describe what near vs not-near means in the commit, and maybe in the ids?